pkg/storage/azure/azure.go

```diff
@@ -486,23 +463,37 @@ func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
 				return err
 			}
 
-			containerName := "image-registry"
+			for i := 0; i < 5000; i++ {
+				// If the bucket name is blank, let's generate one
+				if len(d.Config.Container) == 0 {
+					d.Config.Container = fmt.Sprintf("%s-%s-%s", infra.Status.InfrastructureName, imageregistryv1.ImageRegistryName, strings.Replace(string(uuid.NewUUID()), "-", "", -1))[0:62]
```

Comment:
nit - comment that the container name limit is 63 characters.

Reply:
updated in current push